### PR TITLE
Implement header mobile + desktop 

### PR DESCRIPTION
**Pull request by EthanMoveo**

### CODE REVIEW

File: src/components/atoms/Header.styles.tsx [hunk 1]
```diff
@@ -0,0 +1,122 @@
+import styled from 'styled-components';
+import { colors } from './../../constants/colors';
+
+export const HeaderContainer = styled.header`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  position: relative;
```
Comment (noyflaysher) [inline src/components/atoms/Header.styles.tsx]: why relative and not sticky ?

File: src/components/atoms/Header.styles.tsx [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+  }
+`;
+
+export const LeftSection = styled.div`
+  display: flex;
+  align-items: center;
+`;
+
+export const MenuButton = styled.button`
+  background: none;
```
Comment (noyflaysher) [inline src/components/atoms/Header.styles.tsx:25]: why do you need it ? its the default

File: src/components/atoms/Header.styles.tsx [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+  border: none;
+  cursor: pointer;
```
Comment (noyflaysher) [inline src/components/atoms/Header.styles.tsx]: its the default

File: src/components/atoms/Header.styles.tsx [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+  width: 24px;
+  height: 24px;
+  margin-top: 11px;
+  margin-bottom: 11px;
+  margin-left: 20px;
```
Comment (noyflaysher) [inline src/components/atoms/Header.styles.tsx]: write all the margin in one line

File: src/components/atoms/Header.styles.tsx [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+  width: 24px;
+  height: 24px;
+`;
+
+export const LogoIcon = styled.img`
+  width: 32.93px;
+  height: 32px;
+  margin: 7px 0;
+  
+  position: absolute;
```
Comment (noyflaysher) [inline src/components/atoms/Header.styles.tsx:43]: try not to use position absolute

File: src/components/atoms/Header.styles.tsx [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+
+  }
+`;
+
+export const PlatformName = styled.span`
+  display: none;
+  margin-left: 10px; 
+  font-size: 27px;
+  font-weight: 200;
+  font-family: 'Helvetica Neue';
```
Comment (noyflaysher) [inline src/components/atoms/Header.styles.tsx]: use the fonts variable

File: src/constants/fonts.ts [hunk 7]
```diff
@@ -0,0 +1,93 @@
+export const fonts = {
+    DESKTOP: {
+      H1: {
+        fontFamily: 'Helvetica Neue',
```
Comment (noyflaysher) [inline src/constants/fonts.ts]: use form variable

File: src/components/atoms/Header.styles.tsx [hunk 8]
```diff
@@ -0,0 +1,122 @@
...
+  height: 32px;
+  margin: 7px 0;
+  
+  position: absolute;
+  left: 50%;
+  transform: translateX(-50%);
+  
+  @media (min-width: 1440px) {
+    position: static;
+    transform: none;
```
Comment (noyflaysher) [inline src/components/atoms/Header.styles.tsx:49]: its nessecery?

File: src/components/atoms/Header.styles.tsx [hunk 9]
```diff
@@ -0,0 +1,122 @@
...
+  width: 32.93px;
+  height: 32px;
+  margin: 7px 0;
+  
+  position: absolute;
+  left: 50%;
+  transform: translateX(-50%);
+  
+  @media (min-width: 1440px) {
+    position: static;
```
Comment (noyflaysher) [inline src/components/atoms/Header.styles.tsx:48]: why ?

File: src/components/atoms/Header.styles.tsx [hunk 10]
```diff
@@ -0,0 +1,122 @@
...
+export const HeaderContainer = styled.header`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  position: relative;
+  height: 46px;
+
+  @media (min-width: 1440px) {
+    height: 36px;
+    margin: 14px 127px;
```
Comment (noyflaysher) [inline src/components/atoms/Header.styles.tsx]: fix the margin top we talked about it

File: src/components/atoms/Header.styles.tsx [hunk 11]
```diff
@@ -0,0 +1,122 @@
...
+  align-items: center;
+`;
+
+export const MenuButton = styled.button`
+  background: none;
+  border: none;
+  cursor: pointer;
+  width: 24px;
+  height: 24px;
+  margin-top: 11px;
```
Comment (noyflaysher) [inline src/components/atoms/Header.styles.tsx]: we talked about it

File: src/components/atoms/Header.styles.tsx [hunk 12]
```diff
@@ -0,0 +1,122 @@
...
+  }
+`;
+
+export const MenuIcon = styled.img`
+  width: 24px;
+  height: 24px;
+`;
+
+export const LogoIcon = styled.img`
+  width: 32.93px;
```
Comment (noyflaysher) [inline src/components/atoms/Header.styles.tsx]: 32 px its ok

File: src/components/atoms/Header.styles.tsx [hunk 13]
```diff
@@ -0,0 +1,122 @@
...
+    margin-left: 0;
+    margin-top: 15.5px;
+
+  }
+`;
+
+export const PlatformName = styled.span`
+  display: none;
+  margin-left: 10px; 
+  font-size: 27px;
```
Comment (noyflaysher) [inline src/components/atoms/Header.styles.tsx:55]: use var style

File: src/components/atoms/Header.styles.tsx [hunk 14]
```diff
@@ -0,0 +1,122 @@
...
+  font-weight: 200;
+  font-family: 'Helvetica Neue';
+  line-height: 32.64px;
+  color: #${colors.PRIMARY.black};
```
Comment (noyflaysher) [inline src/components/atoms/Header.styles.tsx]: try to use like this : color:${Color.C....}

File: src/components/atoms/Header.styles.tsx [hunk 15]
```diff
@@ -0,0 +1,122 @@
...
+  margin-top: 15.5px;
```
Comment (noyflaysher) [inline src/components/atoms/Header.styles.tsx]: you dont need it

File: src/components/atoms/Header.styles.tsx [hunk 16]
```diff
@@ -0,0 +1,122 @@
...
+    margin-left: 32px; 
+    margin-top: 15.5px; 
+  }
+`;
+
+export const NavLinkItem = styled.a`
+  text-decoration: none;
+  color: ${colors.PRIMARY.black};
+  font-size: 18px;
+  font-weight: 200;
```
Comment (noyflaysher) [inline src/components/atoms/Header.styles.tsx]: use font variable

File: src/constants/colors.ts [hunk 17]
```diff
@@ -0,0 +1,12 @@
+export const colors = {
+    PRIMARY: {
+      black: '#000000',
+      gray: '#979797',
```
Comment (noyflaysher) [inline src/constants/colors.ts]: change the names of the color to upper letters

Comment (EthanMoveo) [reply src/constants/colors.ts]: Done

File: src/components/atoms/Header.styles.tsx (hunk 16 above)
Comment (EthanMoveo) [reply src/components/atoms/Header.styles.tsx]: Done

File: src/components/atoms/Header.styles.tsx (hunk 15 above)
Comment (EthanMoveo) [reply src/components/atoms/Header.styles.tsx]: Changed margin properties

File: src/components/atoms/Header.styles.tsx (hunk 12 above)
Comment (EthanMoveo) [reply src/components/atoms/Header.styles.tsx]: Done

File: src/components/atoms/Header.styles.tsx (hunk 11 above)
Comment (EthanMoveo) [reply src/components/atoms/Header.styles.tsx]: Fix all margins in the style

File: src/components/atoms/Header.styles.tsx (hunk 10 above)
Comment (EthanMoveo) [reply src/components/atoms/Header.styles.tsx]: Done

File: src/components/atoms/Header.styles.tsx (hunk 8 above)
Comment (EthanMoveo) [reply src/components/atoms/Header.styles.tsx:49]: When using the absolute property yes..

File: src/constants/fonts.ts (hunk 7 above)
Comment (EthanMoveo) [reply src/constants/fonts.ts]: Done

File: src/components/atoms/Header.styles.tsx (hunk 6 above)
Comment (EthanMoveo) [reply src/components/atoms/Header.styles.tsx]: Done

File: src/components/atoms/Header.styles.tsx (hunk 4 above)
Comment (EthanMoveo) [reply src/components/atoms/Header.styles.tsx]: Changed all the margins properties to padding

File: src/components/atoms/Header.styles.tsx (hunk 3 above)
Comment (EthanMoveo) [reply src/components/atoms/Header.styles.tsx]: for cursor yes but for background and border no, background and border are set to gray and 1px black by default

File: src/components/atoms/Header.styles.tsx (hunk 2 above)
Comment (EthanMoveo) [reply src/components/atoms/Header.styles.tsx:25]: by default, button has a grey background

File: src/components/atoms/Header.styles.tsx (hunk 1 above)
Comment (EthanMoveo) [reply src/components/atoms/Header.styles.tsx]: changed it to sticky

File: src/components/atoms/Header.styles.tsx [hunk 18]
```diff
@@ -0,0 +1,108 @@
...
+  
+  @media (min-width: 1440px) {
+    position: static;
+    transform: none;
+  }
+`;
+
+export const PlatformName = styled.span`
+  display: none;
+  font-weight: ${fonts.DESKTOP.H1.fontWeight};
```
Comment (noyflaysher) [inline src/components/atoms/Header.styles.tsx]: use H1 :
ex:
export const ContactDetailsSubTitle = styled.div`
	${css`
		${TitleT15Bold}
	`}
	color:${Color.COLORS.NEUTRAL.FONT_LIGHT};
	font-size: 15px;
	display: flex;
	gap: 4px;
	align-items: center;
`;

File: src/constants/fonts.ts [hunk 19]
```diff
@@ -0,0 +1,95 @@
...
+      fontFamily,
+      fontWeight: '200',
+      fontSize: '24px',
+      color: colors.PRIMARY.BLACK,
+    },
+    button: {
+      fontFamily,
+      fontWeight: '200', 
+      fontSize: '16px',
+      color: '#FFFFFF',
```
Comment (noyflaysher) [inline src/constants/fonts.ts]: use variable

File: src/constants/fonts.ts [hunk 20]
```diff
@@ -0,0 +1,81 @@
...
+      fontSize: '16px',
+      color: colors.PRIMARY.BLACK,
+    },
+    H3: {
+      fontFamily: fontFamilyPrimary,
+      fontWeight: '400', 
+      fontSize: '18px',
+      color: colors.PRIMARY.BLACK,
+    },
+    bodyText: {
```
Comment (noyflaysher) [inline src/constants/fonts.ts]: upper letters

**Review by noyflaysher — APPROVED** [review]

**Pull request merged**